code/modules/mob/living/simple_animal/hostile/vannatusk.dm

```diff
@@ -45,6 +45,53 @@
 
 	psi_pingable = FALSE
 
+/mob/living/simple_animal/hostile/vannatusk/ghostrole
+	name = "vannatusk"
+	desc = "A monstrous interdimensional invader. Its body is protected by a chitin carapace."
+	icon = 'icons/mob/npc/vannatusk.dmi'
+	icon_state = "vannatusk"
+	icon_living = "vannatusk"
+	icon_dead = "vannatusk_dead"
+
```

Comment:
This has a lot of repeated vars from the base type. You only need to change what should actually be different, the rest is inherited.